### DISCUSSION TITLE
Trying to install this extension on a Directus docker instance running on K8S

### DISCUSSION THREAD

**Discussion by marcelhorner** [ChappIO/directus-extension-prometheus#3] [Q&A]
Hello, I am trying to install this extension on a Directus docker instance running on a K8S cluster.

I've tried to connect to the running pod and execute the install command:
`npm install directus-extension-prometheus`

Command executed without errors:
`added 4 packages, removed 2 packages, and audited 5 packages in 11s`

But when accessing "Settings\Extensions", on Directus UI, I can't see any extension.

Hitting /metrics shows this:
`{
    "errors": [
        {
            "message": "Route /metrics doesn't exist.",
            "extensions": {
                "code": "ROUTE_NOT_FOUND",
                "path": "/metrics"
            }
        }
    ]
}`

My question is, should I be able to get metrics at \metrics right after running the npm install command?

Thank you!

**Comment by ChappIO** [comment]
I've designed this package to work when installing into a "managed" directus project, meaning it has to be installed into a package.json that is in the working directory of your directus instance.

The way I create my directus project is:

```
npm init
npm install directus
npm install directus-extension-prometheus
```

Then interact with directus through the CLI
```
npx directus bootstrap
npx directus --help

# etc...
```

If you install the extension this way, it will be automatically detected and loaded.

Note that I do NOT use the directus docker base image. I use the node base image instead and build my own image containing all the files and extensions I need. (Which is why I developed a bunch of devops related extensions).
